Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1,14 +1,933 @@
+
+- PIM API to get role assignments: - /beta/privilegedAccess/azureResources/resources/{uniquePIMIdentifier}/roleAssignments?$expand=subject,roleDefinition($expand=resource)&$filter=(memberType ne '{filterCondition}')<br />
+**Properties:** subject.type, roleDefinition.displayName
+ <br />
+
+- ARM API to list role assignment at scope: - /{scope}/providers/Microsoft.Authorization/roleAssignments?api-version=2018-01-01-preview <br />
+**Properties:** principalType, roleDefinitionId (Role name resolved from roleDefinitionId)
+ <br />
+
+- ARM API to list classic role assignment at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Authorization/classicAdministrators?api-version=2015-06-01<br />
+**Properties:** role
+ <br />
+
+- Graph API to fetch additional details: - /beta/directoryObjects/getByIds?$select=id,userPrincipalName,onPremisesExtensionAttributes,userType,creationType,externalUserState<br />
+**Properties:** userType (To identify guest accounts)
+ <br />
+
+<br />
+
+___ 
+
+## Azure_Subscription_AuthZ_Limit_ClassicAdmin_Count 
+
+### DisplayName 
+Limit access per subscription to 2 or less classic administrators 
+
+### Rationale 
+The v1 (ASM-based) version of Azure resource access model did not have much in terms of RBAC granularity. As a result, everyone who needed any access on a subscription or its resources had to be added to the Co-administrator role. These individuals are referred to as 'classic' administrators. In the v2 (ARM-based) model, this is not required at all and even the count of 2 classic admins currently permitted is for backward compatibility. (Some Azure services are still migrating onto the ARM-based model so creating/operating on them needs 'classic' admin privilege.) 
+
+### Control Settings 
+```json 
+{
+    "NoOfClassicAdminsLimit": 2
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> The count of classic administrators does not exceed 2.
+> 
+> **Failed:** 
+> More than 2 classic administrators accounts found.
+> 
+> **Verify:** 
+> RBAC result not found (sufficient data is not available for evaluation).
+> 
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 You need to remove any 'Classic Administrators/Co-Administrators' who should not be in the role. Please follow these steps: <br />(a) Logon to https://portal.azure.com/ <br />(b) Navigate to Subscriptions <br />(c) Select the subscription <br />(d) Go to 'Access Control (IAM)' and select the 'Classic Administrators' tab. <br />(e) Select the co-administrator account that has to be removed and click on the 'Remove' button. <br />(f) Perform this operation for all the co-administrators that need to be removed from the subscription. 
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list classic role assignment at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Authorization/classicAdministrators?api-version=2015-06-01<br />
+**Properties:** properties.role
```

Comment:
Should this be `[*].properties...`, considering the response is a list of assignments. `[*]` to indicate - for all assignments.